code/src/OSFramework/Feature/Auxiliar/CellStyleInfo.ts

```diff
@@ -0,0 +1,21 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace OSFramework.Feature.Auxiliar {
```

Comment:
The name of this namespace is strange, and we should review this in the future.

Reply:
I agree, even though this namespace is an old one